bundles/framework/featuredata/view/FeatureDataContainer.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Tabs } from '../../../../src/react/components/Tabs';
```

Comment:
Tuolle src/react/components löytyy tällainen lyhempi muoto jota on käytetty:
```suggestion
import { Tabs } from 'oskari-ui';
```